#!/usr/bin/python3


def raise_exception_msg(message=""):
    '''
    This function raises a name exception with a message
    Args:
        message: the message to pass
    Returns:
        None
    '''
    raise NameError(message)
